benefits/core/templates/core/page.html

```diff
@@ -13,7 +13,8 @@
     <div class="row">
         {% if page.image %}
         {% load static %}
-        <div class="col-lg-6 image" style="background: 48% 75% url('{% static page.image.src %}') no-repeat; background-size: cover;">
+        <div class="col-lg-6 image">
+            <img src="{% static page.image.src %}" class="core-image" alt="Rider tapping a debit card to pay for their trip" >
```

Comment:
Maybe this commit didn't make it up yet?

Alt tag needs to be `{% translate "core.index.image" context "image alt text" %}`.

Same as how it's done on the Enrollment success image.